lib/commands/samples.ts

```diff
@@ -49,3 +49,39 @@ class CloneCommandParameter implements ICommandParameter {
 		}).future<boolean>()();
 	}
 }
+
+class PrintCordovaSamplesInformationCommand implements ICommand {
```

Comment:
I'll try to remove most of the duplicate code, but we prefer to have the commands separated (to have separate command for each framework) as it makes it easier to show correct help and information about the framework (context sensitive help).